### ISSUE TITLE
[Feature Request] Allow config the gemfile directory instead of `bundler.d/`

### ISSUE THREAD

**Issue by Whirlwind** [ManageIQ/bundler-inject#38]
I have some projects and they use same gemfiles, but other not.
Now, the `bundler-inject` only search the gemfile in:
```
global - ~/.bundler.d/*.rb
project - $PROJECT_DIR/bundler.d/*.rb
```
I want to set a configuration to custom the `bundler.d/`, eg: `~/ios_project/gemfiles` and `~/ruby_project/gemfiles`.

**Comment by Fryguy** [comment]
@Whirlwind Yeah, I think `eval_gemfile` is more the way to go.  bundler-inject is more about overriding things in existing Gemfiles, but it sounds like your request is more to completely use a different Gemfile base.
